src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = wall.querySelector('.spider');
+const spiderSize = spider.clientWidth;
```

Comment:
it's better to use [offsetWidth](https://learn.javascript.ru/size-and-scroll#offsetwidth-height) for the spider size, it contains the full width of an element (including borders)